### PR TITLE
 Fix layout issues on set assignments and add groups modal 

### PR DESCRIPTION
**Pull request by Scratchcat1**
- Fix touching buttons on narrow screens on the add groups modal.
- Fix unaligned cards on set assignments page

### CODE REVIEW

**Review by Morpheu5 — APPROVED** [review]

File: src/scss/common/assignments.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.assignments-grid {
```
Comment (bh412) [inline src/scss/common/assignments.scss:1]: Keeps the cards the same height but when 'assign/ unassign' clicked all row expands to match

File: src/app/components/pages/SetAssignments.tsx [hunk 2]
```diff
@@ -144,7 +144,7 @@ const Board = (props: BoardProps) => {
     const boardSubjects = determineGameboardSubjects(board);
     const boardLevels = determineGameboardLevels(board);
 
-    return <Card className="board-card">
+    return <Card className="board-card h-100">
```
Comment (bh412) [inline src/app/components/pages/SetAssignments.tsx:147]: Not sure we want this to be the set height here

File: src/app/components/elements/modals/GroupsModalCreators.tsx [hunk 3]
```diff
@@ -59,23 +59,23 @@ export const groupInvitationModal = (firstTime: boolean) => {
         body: <CurrentGroupInviteModal firstTime={firstTime} />,
         buttons: [
             <RS.Row key={0}>
-                <RS.Col>
+                <RS.Col className="my-2">
```
Comment (bh412) [inline src/app/components/elements/modals/GroupsModalCreators.tsx:62]: I think this modal is actually fine, and it's the 'GameboardCreatedModal' that needed work, but I've done this change to it

**Review by bh412 — COMMENTED** [review]
Mostly works but set assignments has the expanding card quirk so going to move this to won't fix too minor - A nice implementation though and I'll see if there are any other card bits it should be applied to

**Comment by bh412** [comment]
We've since accomplished this in another way so this is redundant.

**Pull request closed without merging**